include/deal.II/base/data_out_base.h

```diff
@@ -2552,8 +2552,11 @@ class DataOutInterface
    * fields.
    */
   virtual
-  std::vector<std_cxx11::tuple<unsigned int, unsigned int, std::string> >
-  get_vector_data_ranges () const;
+  inline std::vector<std_cxx11::tuple<unsigned int, unsigned int, std::string> >
+  get_vector_data_ranges() const
+  {
+    return std::vector<std_cxx11::tuple<unsigned int, unsigned int, std::string> >();
+  }
```

Comment:
But can we move the implementation out of the class and to the bottom of the file as we usually do?